### PR TITLE
github: add an issue template

### PR DESCRIPTION
**Pull request by obnoxxx**
Signed-off-by: Michael Adam <obnox@redhat.com>

### CODE REVIEW

**Comment by obnoxxx** [comment]
I think it's a good idea to have an issue template that helps structure writing issues.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+### Information about the enviroment:
+
+- Heketi version used (e.g. v6.0.0 or master): 
+- Operating system used: 
+- Heketi compiled from sources, as a package (rpm/deb), or container: 
+- If container, which container image: 
+- Using kubernetes, openshift, or direct install: 
+- If kubernetes/openshif, gluster running in kube/opebshift or outside: 
+- More information:
```
Comment (raghavendra-talur) [inline .github/ISSUE_TEMPLATE.md]: We need to also probably ask for deployment method. Custom deployment config or gk-deploy.

Comment (obnoxxx) [reply .github/ISSUE_TEMPLATE.md]: right, i'll add that (and fix the openshift typos...) ;-)

**Comment by raghavendra-talur** [comment]
Just putting out the possibility: 
> If you’re worried about the added clutter in the root directory of your project, we also added support for a .github/ folder. You can put CONTRIBUTING.md, ISSUE_TEMPLATE.md, and PULL_REQUEST_TEMPLATE.md files in .github/ and everything will work as expected.

More than the hiding part, I like group these files under a dir which explains their role.

**Comment by obnoxxx** [comment]
updated with typofix and ask for deployment method

**Comment by obnoxxx** [comment]
> Just putting out the possibility:
> 
> >  If you’re worried about the added clutter in the root directory of your project, we also added support for a .github/ folder. You can put CONTRIBUTING.md, ISSUE_TEMPLATE.md, and PULL_REQUEST_TEMPLATE.md files in .github/ and everything will work as expected.
> 
> More than the hiding part, I like group these files under a dir which explains their role.

If you look at the patch, that's exactly where I put it... :-)

**Comment by obnoxxx** [comment]
@phlogistonjohn agreed, and updated. Thanks for calling out my laziness! :-)

**Pull request merged**